tests/commands/env/list.spec.ts

```diff
@@ -22,7 +22,7 @@ import { BalenaAPIMock } from '../../nock/balena-api-mock';
 import { runCommand } from '../../helpers';
 import { randomBytes } from 'node:crypto';
 
-describe('balena envs', function () {
```

Comment:
@myarmolinsky I maybe just realized now but I think it would be fair to have one test copied with `envs` (maybe the most simple of them) just so we are sure the alises are working and we remove the test when remove the alias